README.md

```diff
@@ -128,25 +251,37 @@ CDAClient client =
         .build();
 ```
 
-Apart from the configuration option, you can use the SDK without modifications with one exception: you need to obtain a preview access token, which you can get in the "API" tab of the Contentful app. In preview mode, data can be invalid, because no validation is performed on unpublished entries. Your app needs to deal with that. Be aware that the access token is read-write and should in no case be shipped with a production app.
+The _Preview Access Token_<sup><a href="https://www.contentful.com/developers/docs/references/content-preview-api/#/introduction/preview-api-authentication" title="A password for this specific API."/>i</a></sup> is exposed on the [Contentful Web App](https://app.contentful.com/deeplink?link=api). 
+
+> Note: In Preview Resources can be invalid since no validation is performed prior to publishing.
+
+Sync
+----
+
+Fetching all Resources and retrieving only changes on subsequent calls is accomplished by using the `.sync()`-methods:
+
+```java
+SynchronizedSpace space = client.sync().fetch();
+```
+
+The _SynchronizedSpace_ will contain _all_ _published_ Resources. If `.preview()` ([see Preview](#preview)) is used it'll also contain all unpublished Resources.
```

Comment:
|Before|After|
|------|-------|
|The _SynchronizedSpace_ will contain _all_ _published_ Resources. If `.preview()` ([see Preview](#preview)) is used it'll also contain all unpublished Resources.| The _SynchronizedSpace_ contains _all_ _published_ Resources. If `.preview()` ([see Preview](#preview)) is used,  it also contains all unpublished Resources.|